ocp_resources/trustyai_service.py

```diff
@@ -0,0 +1,62 @@
+from ocp_resources.resource import NamespacedResource, MissingRequiredArgumentError
+from typing import Optional, Any, Dict
+
+
+class TrustyAIService(NamespacedResource):
+    """
+    https://github.com/trustyai-explainability/trustyai-service-operator/blob/main/api/v1alpha1/trustyaiservice_types.go
+    """
+
+    api_group: str = NamespacedResource.ApiGroup.TRUSTYAI_OPENDATAHUB_IO
+
+    def __init__(
+        self,
+        storage: Optional[Dict[str, Any]] = None,
+        data: Optional[Dict[str, Any]] = None,
+        metrics: Optional[Dict[str, Any]] = None,
+        replicas: Optional[int] = None,
+        image: Optional[str] = None,
+        tag: Optional[str] = None,
+        **kwargs,
+    ) -> None:
+        """
+        TrustyAIService object
+
+        Args:
+            storage (Optional[Dic[str, Any]]): TrustyAIService storage config (format, folder and size).
+            data (Optional[Dict[str, Any]]): TrustyAIService data config (filename and format).
+            metrics (Optional[Dict[str, Any]]): TrustyAIService metrics config.
+            replicas (Optional[int], default: 1): Number of replicas for the TrustyAIService.
```

Comment:
please remove the default value as it is not set in the wrappers' code but by the api